Dockerfile

```diff
@@ -19,36 +18,30 @@ RUN echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ wheez
 RUN pip install awscli --upgrade
 
 # Install Terraform.
-RUN TF_VERSION="0.11.11"; \
+RUN TF_VERSION="0.12.1"; \
     wget https://releases.hashicorp.com/terraform/${TF_VERSION}/terraform_${TF_VERSION}_linux_amd64.zip && \
     unzip terraform_${TF_VERSION}_linux_amd64.zip -d /bin && \
     rm -f terraform_${TF_VERSION}_linux_amd64.zip
 
-# Install ansible
-RUN pip install ansible --upgrade
-
-# We need to build ct_config terraform provider from sources, because of two things:
-# 1. it's still not avaialble in terraform repos [1].
-# 2. latest binary release v0.2.1 is too old and does not support passwd groups.
-#
-# Links:
-# 1 - https://github.com/coreos/terraform-provider-ct/issues/21
-
-# Install Go 1.10.
+# Install Go 1.11.
```

Comment:
No Go 1.12 available yet? Current stable upstream is `1.12.5`.

Reply:
I had some issues during builds with go 1.12, so set to 1.11. Tested with that version locally. So for the current branch of gotemplate provider it is fine